packages/cli/agent-chat/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
We shouldnt have a sub package json

Reply:
Done! moved the agent chat deps to cli root